### ISSUE TITLE
aws_lakeformation_permissions throws "Error: error reading Lake Formation permissions: no permissions found"

### ISSUE THREAD

**Issue by akuzni2** [hashicorp/terraform-provider-aws#19221]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version

TF: 0.13.6
AWS: 3.38

### Affected Resource(s)

aws_lakeformation_permissions

### Terraform Configuration Files

```hcl
resource "aws_lakeformation_permissions" "lakeformation_permissions" {
  principal = "arn:aws:iam::${var.aws_account}:role/my-role"
  permissions = ["ALTER", "SELECT", "DESCRIBE"]
  permissions_with_grant_option = ["ALTER", "SELECT", "DESCRIBE"]

  table {
    database_name = "my_db"
    wildcard      = true
  }

}
```

### Debug Output

```
Error: error reading Lake Formation permissions: no permissions found

  on iam.tf line 69, in resource "aws_lakeformation_permissions" "lakeformation_permissions":
  69: resource "aws_lakeformation_permissions" "lakeformation_permissions" {


exit status 1
```

### Expected Behavior

Terraform should find the permissions defined in the configuration.

### Actual Behavior

Got errors that it couldn't find the permissions.

### Steps to Reproduce

1. `terraform apply`

### Other information
- The same configuration file shown above works with AWS provider version 3.32
- supposedly this was solved as shown in this issue https://github.com/hashicorp/terraform-provider-aws/issues/18034 - however it doesn't seem to work

**Comment by chrisbulgaria** [comment]
Yes, think this is the same as https://github.com/hashicorp/terraform-provider-aws/issues/18446
But surprised me as well that after several lakeformation related fixes  in 3.38 this one is still open.
Maybe add your vote there such we get the fix merged .

**Comment by akuzni2** [comment]
@chrisbulgaria  - got it - closing this issue then since that one is open for the same thing.
